### DISCUSSION TITLE
Scale with Django or Flask?

### DISCUSSION THREAD

**Discussion by DutyCrafterHD** [telekom/scale#2283] [General]
Hi,
can i use scale offline with the Django Framework or with Flask without cdn?

**Comment by W1ckh3ad** [comment]
Hello @DutyCrafterHD,

I've got a solution, a bit hacky but it works.

Install it with npm

Import or Copy `\node_modules\@telekom\scale-components\dist\scale-components` folder to your public or static directory and import the ressources from there.

It has the `esm` and `css` file for the `<script>` and `<link />`

I have tried downloading from the cdn, but you need all the files (including the bundled files), and they get probablya new name on each build.

**Reply by DutyCrafterHD** [reply]
Hi @W1ckh3ad ,
i got the same idea but got an error. I forgot the type="module" in the script tag. Now it is working. Thank you
